src/Deposit.js

```diff
@@ -523,6 +533,21 @@ export default class Deposit {
     // Bitcoin txHash; no verification initially.
   }
 
+  /**
+   * Registers a handler for notification when the Bitcoin funding transaction
+   * has received a confirmation
+   *
+   * @param {OnReceivedConfirmationHandler} onReceivedConfirmationHandler
+   *        A handler that passes an object with the transactionID and
+   *        confirmations as its parameter
+   */
+  onReceivedConfirmation(onReceivedConfirmationHandler) {
```

Comment:
Thoughts on clarifying the terminology in this class as "funding transaction confirmation" for the docs and function and variable names? It's long winded, but since the `Deposit` class also touches redemption, being clear will be useful I think.

Reply:
I was debating that myself but went for the shorter name. I figured we could talk about it in review :D

Renamed to `onReceivedFundingConfirmation` since we a getter called `fundingConfirmation` and also it is a bit shorter than `onReceivedFundingTransactionConfirmation`. I guess we could go with `onFundingTransactionConfirmation` or `onFundingConfirmation` maybe too? Let me know your preference. I think it is also little tricky to name because it's when the transaction receives a single confirmation vs all required confirmations.